### ISSUE TITLE
Add support for Dashboard overlays

### ISSUE THREAD

**Issue by LibreVR** [Hotrian/OpenVRDesktopDisplayPortal#4]
I haven't tested this application yet, but I noticed one thing from your description:

> Note: Overlays will always draw ontop of other game geometry. This is less noticable if you attach them to the controller since things rarely come between you and the controllers. Otherwise you may want to attach them to the screen, or put them on a wall behind you or up in the sky.

Did you know that there is also a type of overlay called a Dashboard overlay? Dashboard overlays will show up as a tab in the SteamVR dashboard, so it is much more convenient when you don't want the window in front of you all the time.

**Comment by Hotrian** [comment]
Yes, I did know about these dashboard overlays, but at that point it's basically the same thing as the SteamVR Desktop tab :P

I'll probably add support for Dashboard Overlays in the future, but there are a few other things that I think might be a bit more important before then.

Edit: If anyone does want support for Dashboard Overlays though, go ahead and comment below. If there is enough demand for these I will begin supporting them ASAP, otherwise I will add it when I have a chance anyway.

**Comment by Hotrian** [comment]
Also I think it is not immediately clear from the current readme, but these are not static overlays that are stuck to the screen or to a position in the world (though you cannot currently move them from VR). Instead, you can attach these to your controllers like my OpenVRTwitchChat

(really old picture)
![demo](https://camo.githubusercontent.com/1363683a0c0223f37a70d3b21be3dec9412f3c7b/68747470733a2f2f7468756d62732e6766796361742e636f6d2f53696e66756c486f6e65737447656e65742d73697a655f726573747269637465642e676966)

The overlays shouldn't be visible from the backside, so you can put them on the back of your controller and look at them whenever you want to, and you can make them transparent when you're not looking at them:

![demo2](https://cloud.githubusercontent.com/assets/12481327/16932625/891983d0-4d15-11e6-9d0b-158e9b3b10c4.gif)

Or if you want to, you _can_ attach them to the screen or drop them in the world. 

So yeah, I do intend to support putting them in the Dashboard tab eventually, but right now I think this covers most use cases.

Also, [this old video on youtube](https://www.youtube.com/watch?v=679pAcmnRWA) shows the three modes currently supported (World, Screen, & Controller Attached), and this project uses the same system, though there have been a number of changes/upgrades since then :).
